src/index.html

```diff
@@ -11,7 +11,165 @@
+          <input 
+            type="checkbox" 
+            name="I accept" 
+            id="IAccept"
+            autocomplete="off" 
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend class="legend">Registration</legend>        
+        <label class="label-top">
+            E-mail: 
+          <input 
+            type="email" 
+            placeholder="email@example.com"
+            name="email"
+            id="email" 
+            required
+          >   
+        </label>
+        <label class="label">
+            Password:
+          <input 
+            type="password" 
+            name="pasword" 
+            id="password"
+            required
+          >
+        </label> 
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="label-top">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="cats" 
+              value="Yes"
+              id="Yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              type="radio" 
+              name="cats" 
+              value="No"
+              id="No"
+            >
+            No 
+          </label>
+        </div>    
+        <label class="label">
+            What is your favorite color? 
+          <input 
+            type="color" 
+            name="favorite color"
+            id="favoriteColor"
+          >
+        </label>     
+        <label class="label">
+            What time do you go to bed?
+          <input 
+            type="time"
+            step="1"
+            name="time to sleep"
+            id="timeToSleep"
+          >
+        </label>     
+        <label class="label">
+             What are yours favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>          
+        <label class="label">
+            How do you rate our work?
+          <input 
+            type="range" 
+            min="1" 
+            max="10" 
+            name="Our work"
+            default value="1"
+            id="ourWork"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend class="legend">Additional info:</legend>
+        <label class="label-top">
+            Comments:
+          <textarea 
+            name="Comments"
+            maxlength="80" 
+          >
+          </textarea>
+        </label>
+        <label class="label">
+            Would you recommed us?
+          <select name="recommed">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
```

Comment:
```suggestion
        <label class="label">
          Would you recommed us?
          <select name="recommed">
            <option value="yes">yes</option>
            <option value="no">no</option>
          </select>
        </label>
```
Nested element (text): +2 space indentation